beater/request/result.go

```diff
@@ -88,7 +88,7 @@ var (
 		IDResponseErrorsInvalidQuery:       {Code: http.StatusBadRequest, Keyword: "invalid query"},
 		IDResponseErrorsDecode:             {Code: http.StatusBadRequest, Keyword: "data decoding error"},
 		IDResponseErrorsValidate:           {Code: http.StatusBadRequest, Keyword: "data validation error"},
-		IDResponseErrorsMethodNotAllowed:   {Code: http.StatusMethodNotAllowed, Keyword: "only POST requests are supported"},
+		IDResponseErrorsMethodNotAllowed:   {Code: http.StatusMethodNotAllowed, Keyword: "method not supported"},
```

Comment:
similarly to this, L85 https://github.com/elastic/apm-server/pull/3004/files#diff-f5ea74122372bd20b6b1d9543d5123e6R85 message could be replaced with a "Not authorized" or similar, since we will be using tokens and/or keys